integration-auth/src/test/java/org/symphonyoss/integration/authentication/AuthenticationContextTest.java

```diff
@@ -0,0 +1,137 @@
+/**
+ * Copyright 2016-2017 Symphony Integrations - Symphony LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.symphonyoss.integration.authentication;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.symphonyoss.integration.exception.RemoteApiException;
+
+/**
+ * Unit test for {@link AuthenticationContext}
+ * Created by Evandro Carrenho on 21/04/17.
+ */
+@RunWith(MockitoJUnitRunner.class)
+public class AuthenticationContextTest {
```

Comment:
These tests don't seem to have anything to do with the code changes in AuthenticationContext above.